addon/run-task.ts

```diff
@@ -9,7 +9,8 @@ import {
   EmberRunTimer,
   EmberRunQueues,
 } from './types';
-import { registerDisposable } from './utils/disposable';
+import { registerDestructor } from '@ember/destroyable';
+import { Destructor } from 'ember-destroyable-polyfill/-internal/destructors';
```

Comment:
✂️

Reply:
Err. Cut which? We need the destroyable import.